src/ui/Windows/PasswordPolicyDlg.cpp

```diff
@@ -190,9 +197,13 @@ BOOL CPasswordPolicyDlg::OnInitDialog()
   // If started with Tall and won't fit - return to be called again with Wide
   if (m_bLongPPs && !GetMainDlg()->LongPPs(this)) {
     EndDialog(-1);
-    return FALSE;
+    return TRUE;
```

Comment:
This commit corrects and documents the return code from OnInitDialog throughout PWS. The return code from OnInitDialog is to indicate where focus is to be placed (TRUE - default, FALSE - it has been changed) - it does not indicate whether the dialog is shown. In this case, the focus wasn't changed and so the correct return code is TRUE.  In reality, it doesn't matter in this case as the call to EndDialog prevents the dialog ever being shown.